wx_test_paste_census.cpp

```diff
@@ -47,6 +51,32 @@
 namespace
 {
 
+// Helper for building the diagnostic message.
```

Comment:
Why has this moved around? Could you please put it back to avoid unnecessary changes in the diff?